certified-connectors/absentify/README.md

```diff
@@ -0,0 +1,23 @@
+# absentify
+
+absentify is free and specially designed for the Microsoft 365 environment to manage your team's or employee's absences and have an overview of all time off and reason given. You benefit from full integration of paid time off tracking with Microsoft Teams and Microsoft 365, no more boring Excel sheets. absentify is the easiest way to approve vacation time and holidays.
+
```

Comment:
@marc-on-github please update the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

Reply:
fixed